luacode/formatter.lua

```diff
@@ -477,7 +471,7 @@ end
 -- @usage indentCode('local var', '\n', true, '\t')
 -- @usage indentCode('local var', '\n', true, --[[tabulationSize]]4, --[[indentationSize]]2)
 --------------------------------------------------------------------------------
-function M.indentcode(source, delimiter,indenttable, ...)
+function M.indentcode(source, delimiter,indenttable, platform, ...)
```

Comment:
What does this `platform` arg do? Is it vestigial?

Reply:
Nothing anymore.  It was needed so we could select between the `.bat` and .sh` scripts